molecule_docker/playbooks/create.yml

```diff
@@ -128,6 +128,7 @@
         privileged: "{{ item.privileged | default(omit) }}"
         security_opts: "{{ item.security_opts | default(omit) }}"
         devices: "{{ item.devices | default(omit) }}"
+        links: "{{ item.links | default(omit) }}"
```

Comment:
Is this field available in the `molecule.yml` and missing here? A test would be nice or it already exists?

Reply:
We can absolutely make a test for this!